src/main/scala/edu/twitter/classification/Classifier.scala

```diff
@@ -30,16 +29,20 @@ class Classifier(ssc: StreamingContext) {
   def createClassifiedStream(modelName: String): DStream[ClassifiedTweet] = {
     val tweets = new TwitterStream(ssc).createStream()
     val supportedLangIso = Set("en", "eng")
-    val dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss")
+    val dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
     val classifiedStream = for {
       tweet <- tweets
       if supportedLangIso(tweet.getLang)
-      date = dateFormat.format(new Date())
       resOption = ModelClient.callModelService(modelName, tweet.getText)
       if resOption.isPresent
       res = resOption.get
-    } yield ClassifiedTweet(res.getLabel, tweet.getText, date)
+      label = if (res.getLabel == 0) -1 else 1
```

Comment:
I guess now we reached a confusion in the `label` representation. I guess it would be better represented as a string instead of numeric values. maybe `Enumeration`. I guess it is up to you to use the `String/Enumeration` representation here or changing the `GenericModel` API to support the cleaner way.

Reply:
Kibana needs the value as a double to aggregate label according to the location,
if changed to string or enum we will lose the location view on the map.

Reply:
Ok, but could you please at least unify the notation, meaning change the representation of `happy` and `sad` across the project. I can help if you wish.